packages/fluentui/react-northstar-prototypes/src/prototypes/instructionMessage/instructionMessage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useFluentContext } from '@fluentui/react-bindings';
+import { Box, Ref } from '@fluentui/react-northstar';
+
+interface InstructionMessageProps {
+  message: string;
+  role?: string;
+  'aria-label'?: string;
+}
+
+const InstructionMessage: React.FunctionComponent<InstructionMessageProps> = props => {
+  const { children, message, ...rest } = props;
+
+  const narrationHappened = React.useRef(false);
+  const instructionRef = React.useRef<HTMLElement>();
+  const timeoutMessageNarrate = React.useRef<number>();
+  const timeoutMessageRemove = React.useRef<number>();
+  const context = useFluentContext();
+  const areaFocusableElements = () =>
```

Comment:
Could we get it from props as well? Like if the consumer wants to have the messages on buttons, but not on inputs/selects?

Reply:
I am not sure if I get the suggestion